src/hooks/useViewportSizes/index.jsx

```diff
@@ -0,0 +1,29 @@
+import { useMediaQuery } from "usehooks-ts";
```

Comment:
Well tested, small hook that includes a lot of nice things, like listening to the MediaQueryList `onchange` events instead of throttling browser resize, among other things